baybe/searchspace/continuous.py

```diff
@@ -118,11 +122,24 @@ def from_dataframe(
                 is created with default optional arguments. For more details, see
                 :func:`baybe.parameters.utils.get_parameters_from_dataframe`.
 
+        Raises:
```

Comment:
This is kind of redundant since we only have this type of continuous parameter right? But I guess mypy still complains? Still makes sense, just want to make sure :)

Reply:
Yes, the issue is that our current class hierarchy of the continuous parameters is somewhat "useless" because we only have one type. For logical reasons, the function should accept the base class, but inside we currently use stuff that is only defined for the subclass. Hence, the `isinstance` check is needed and thus the `Raises` section. Once the class hierarchy gets an additional member, we have to rethink stuff. But for now, this is the easiest solution (and we cannot forget to remove the check once it's no longer needed)